packages/opentelemetry/README.md

```diff
@@ -59,4 +65,18 @@ await tracer.startActiveSpan('span.name', async (span) => {
 });
 ```
 
-When using code from the OpenTelemetry libraries, make sure you import it via `@prairielearn/opentelemetry` instead of installing it separately to ensure that there is only one version of each OpenTelemetry package in use at once. If the desired functionality is not yet exported, please add it!
+## Metrics
+
+You can also use the `instrumentedWithMetrics` helper to automatically capture a duration histogram and error count:
+
+```ts
+import { instrumentedWithMetrics } from '@prairielearn/opentelemetry';
+
+await instrumentedWithMetrics('operation.name', async () => {
+  const random = Math.random() * 1000;
+  await new Promise((resolve) => setTimeout(resolve, random));
+  if (random > 900) {
+    throw new Error('Failed!');
+  }
+});
+```
```

Comment:
I really like the `instrumentedWithMetrics()` wrapper. How are the [buckets](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#explicit-bucket-histogram-aggregation) being specified for the underlying histogram?

Reply:
They default to the following ([source](https://github.com/open-telemetry/opentelemetry-js/blob/47444f2a4c13a80f3328ffafc5b4c8db9e9a0886/packages/sdk-metrics/src/view/Aggregation.ts#L105)):

```ts
[0, 5, 10, 25, 50, 75, 100, 250, 500, 1000]
```

Unfortunately, configuring them is kind of a pain; see https://github.com/open-telemetry/opentelemetry-js/issues/2999#issuecomment-1432337849. We can still do it, of course; it's just super unfortunate that the boundaries have to be described in a completely different part of the app.

Reply:
Oh yuck! In practice we'll probably be ok with those default boundaries for most of the timing we are doing, but this is really clunky.

Reply:
Yeah, it's not great. I opened https://github.com/open-telemetry/opentelemetry-js/issues/3619 to discuss this problem specifically.